### ISSUE TITLE
another notification requirement: x has just shared project named xyz with you

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#998]
Reminded by the two new issues Fay has added about notifications etc this is another one I was asked about e.g. "Would it be possible to be notified via an alert of some sort that someone has shared a project (or projects) with me?"

**Comment by torinfo** [comment]
In general, this is not that easy in Xerte. Xeret does not know the email adresses of it's users.

The email address that Xerte tries to use is the login name combined with the email domain set in the management page. If that happens to be the correct email address, we can send an email to the user.

I think we should enable useres to set their own notification emailadress and also enable them to enable/disable notifications.

**Comment by ronm123** [comment]
@torinfo when I was asked about this I did mention that most Xerte installs don't have user email details but in fact the question was about a pop-up alert when logging in rather than an alert sent by email. e.g. a user with lots of projects may not spot a new project shared with them and apparently projects are often shared but the person sharing doesn't always send details via email.

**Comment by FayCross** [comment]
Could this be done with something like a star icon next to projects in the workspace that have been shared with you but you haven't viewed yet?

**Comment by ronm123** [comment]
@FayCross I did wonder about some kind of visual change to newly shared projects. Could just be the name in a different colour but yes an icon of some sort might be better.
Then again combining the two other enhancements that you mentioned earlier perhaps a pop up window that shows a history of recent changes e.g. list of projects deleted, shared, updated etc.
Perhaps something to discuss during the next dev day.

**Comment by torinfo** [comment]
@ronm123 @FayCross Yes, let's discuss this on the dev day. Notification by email (if the email is available) does not require a database change. All the other proposals need some kind of administartion in the database.
